Expo1900/Expo1900/Controller/EntryDetailViewController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  EntryDetailViewController.swift
+//  Expo1900
+//
+//  Created by Minsup, Jusbug on 2023/06/28.
+//
+
+import UIKit
+
+final class EntryDetailViewController: UIViewController {
+    var entry: Entry?
+    @IBOutlet weak private var entryDescriptionLabel: UILabel!
+    @IBOutlet weak private var entryImageView: UIImageView!
+    static let identifier = "entryDetailViewController"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationItem.title = self.entry?.name
+        self.configureComponents()
+    }
+    
+    private func configureComponents() {
+        if let entry = self.entry {
```

Comment:
swift 5.7부터 아래와 같이 축약할 수 있습니다.
```suggestion
        if let entry {
```

Reply:
좋은 정보 감사합니다.!!